client/errors.go

```diff
@@ -112,6 +112,19 @@ func classifyError(err error, fallbackType diag.Type, accounts []Account, opts .
 					)...),
 				),
 			}
+		case "UnsupportedOperation":
+			if strings.Contains(ae.ErrorMessage(), "The functionality you requested is not available in this region.") {
+				return diag.Diagnostics{
+					RedactError(accounts, diag.NewBaseError(err,
+						diag.RESOLVING,
+						append(opts,
+							diag.WithType(diag.RESOLVING),
+							diag.WithSeverity(diag.IGNORE),
```

Comment:
if we put this in ignore users won't see this. cc @bbernays

Reply:
I am not sure which type is better as well. My thoughts are next: the case we have here is not fixable by user so we don't need to notify them. For example if user fetches all regions they will always have warnings that they can't fix.

Reply:
I agree with @amanenk. If the user cannot fix the issue then why notify them. The only scenario I can see this not being the behavior we want is if the user specifies only a single region where the resource is not available. In that one case I would expect that the user would just remove the resource from the fetch list